lib/src/components/NxList/NxListText.tsx

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { forwardRef } from 'react';
+import classnames from 'classnames';
+import {NxListTextProps, nxListTextPropTypes} from './types';
+
+const NxListText = forwardRef<HTMLSpanElement, NxListTextProps>((props: NxListTextProps, ref) => {
```

Comment:
`withClass`... maybe.  If it supported refs I'd say definitely.  I'm trying locally to make with work with refs but the type signatures are dfficult

Reply:
> `withClass`... maybe. If it supported refs I'd say definitely. I'm trying locally to make with work with refs but the type signatures are dfficult

Converted to use `withClass`.
88365303aa6c6cd441566999a777569fa0022ec3